Userland/Utilities/sed.cpp

```diff
@@ -831,6 +831,11 @@ static ErrorOr<CycleDecision> apply(Command const& command, StringBuilder& patte
         auto replacement_made = result != pattern_space_sv;
         pattern_space.clear();
         pattern_space.append(result);
+        if (replacement_made && s_args.output_filepath.has_value()) {
+            auto output_file = TRY(Core::File::open(s_args.output_filepath.value(), Core::File::OpenMode::Write | Core::File::OpenMode::Append));
+            TRY(output_file->write_until_depleted(TRY(pattern_space.to_byte_buffer())));
```

Comment:
nit: i would've put these 5 lines into a `write_pattern_space(filepath, pattern_space)` function that's used by both match w and sub w

Reply:
There is an existing `write_pattern_space()` function but it takes different arguments.